CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 Changelog
 ====
+### 1.6.9
+
+* Allow unauthenticated usage of "datasets download", "datasets files".
+  * This will only work after April 8th, 2024. More more details, see:

```

Comment:
Note: per PF and email announcement, we're enabling this behavior on Monday, April 8th, 2024.

I'm hoping to get this merged and a client release tested / deployed ahead of time to de-risk the launch on Monday.

Reply:
suggestion - my preference is that versioning bumps would be in their own PR.  This makes it easier to see where releases are being cut, and also that everything in a release is reflected in the notes.  Not a big deal, wdyt?

Reply:
sg. so for now, i'll revert the versioning bumps but keep the code changes. then later we'll update the changelog on the release.

Reply:
actually, i think i misunderstood the suggestion. this *was* meant to be all the changes in a release, so reframing this PR as a release of 1.6.9 instead.

Reply:
nvm the nvm, chatted w/Jim and we're proposing a standardized flow.

code changes separate from releases, since you could have incremental code changes before you decide to cut a release.

we're okay with the version being outdated until we cut a release (and then we bump then).

going to document this internally, and reframe this PR back to being "Code changes only"